vendor/vendor.json

```diff
@@ -81,12 +81,6 @@
 			"revisionTime": "2015-12-06T04:24:12Z"
 		},
 		{
-			"checksumSHA1": "pWIsEUE53NffjS1goRml43Cvleg=",
```

Comment:
Oh I was expecting that we'd have only a vendor down there under cmd - is that not the case?